### ISSUE TITLE
Fails if attribute is named 'set'

### ISSUE THREAD

**Issue by jgutix** [mrodrig/json-2-csv#185]
## Background Information
- Module Version: ` 3.11.1`
- Node/Browser Version: ` 14`

The issue I'm reporting is with:
- [x] json2csv
- [ ] csv2json

I have...
- [x] searched to see if an issue has already been reported.
- [x] verified that my JSON/CSV data is valid (using something like http://jsonlint.com or https://csvlint.io/).
- [ ] tried upgrading to the latest version of json-2-csv (since the issue may already be fixed).

## Expected Behavior
Should allow field name/attribute to be name "set"

## Actual Behavior
Will throw error:
`TypeError: fieldValue.includes is not a function`

## Data Sample 
CSV:
```csv

```

(or)

JSON:
```json
[{
	"set": "test",
	"otherfieldname": "test2"
}]
```

## Code Example
```javascript
import converter from 'json-2-csv';

const csv = await converter.json2csvAsync([{set: 'test', otherfieldname: 'test2'}]);
console.log(csv)


```

**Comment by mrodrig** [comment]
Thanks for reporting this @jgutix! I found the issue and have a pull request open for the fix. I'm going to see if I can push a release out later today with this and hopefully a fix for another issue that was reported. I'll comment with the version number once it goes out.

**Comment by mrodrig** [comment]
Thanks again @jgutix. I just released the fix in version `3.12.0` which should be available on NPM now.
